keras_cv/models/backbones/resnet_v2/resnet_v2_backbone.py

```diff
@@ -320,9 +254,21 @@ def apply_stack(
 
 
 @keras.utils.register_keras_serializable(package="keras_cv.models")
-class ResNetV2(keras.Model):
+class ResNetV2Backbone(Backbone):
     """Instantiates the ResNetV2 architecture.
 
+    Reference:
+        - [Identity Mappings in Deep Residual Networks](https://arxiv.org/abs/1603.05027) (ECCV 2016)
+
+    The difference in Resnet and ResNetV2 rests in the structure of their
+    individual building blocks. In ResNetV2, the batch normalization and
+    ReLU activation precede the convolution layers, as opposed to ResNetV1 where
+    the batch normalization and ReLU activation are applied after the
+    convolution layers.
+
+    For transfer learning use cases, make sure to read the [guide to transfer
```

Comment:
Break the markdown link after `(` (otherwise it won't render)

Reply:
Moved to one line